lib/metrics.coffee

```diff
@@ -48,6 +49,15 @@ module.exports =
       return if eventName of IgnoredCommands
       Reporter.sendCommand(eventName)
 
+    grim.on 'updated', (deprecation) ->
```

Comment:
@kevinsawicki is this the right way to get the package information? I'd rather not cycle through the deps, but use the info as they come in.